erdjs/src/transaction.ts

```diff
@@ -15,8 +15,8 @@ import {ProtoSerializer} from "./proto";
 
 const createTransactionHasher = require("blake2b");
 
-const TRANSACTION_VERSION = new TransactionVersion(1);
-const TRANSACTION_OPTIONS = new TransactionOptions(0);
+const TRANSACTION_VERSION = TransactionVersion.withDefaultVersion();
+const TRANSACTION_OPTIONS = TransactionOptions.withDefaultOptions();
```

Comment:
Suggest renaming the consts to `DEFAULT_TRANSACTION_VERSION` and `DEFAULT_TRANSACTION_OPTIONS`, if it makes sense for their use.

Reply:
done